examples/toy_domain_adaptation/main.py

```diff
@@ -0,0 +1,92 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import seaborn as sns
+from sklearn.datasets import make_blobs
+from sklearn.linear_model import RidgeClassifier
+from sklearn.metrics import accuracy_score
+from sklearn.preprocessing import OneHotEncoder
+
+from kale.pipeline.multi_domain_adapter import _CoIRLS
+
+
+def plot_scores_dist(ys_score, yt_score, title=None):
```

Comment:
Is this reusable for problems of the same nature? If yes, then it should go to core API.